src/org/labkey/test/components/domain/DomainFormPanel.java

```diff
@@ -175,7 +175,7 @@ public DomainFieldRow manuallyDefineFields(String name)
     public DomainFieldRow manuallyDefineFields(FieldDefinition fieldDefinition)
```

Comment:
These methods are named thus because they start by clicking "define fields manually" (as opposed to uploading a file to define the fields). They are only valid for a new domain. I'll add some documentation.